mm/flow.py

```diff
@@ -53,8 +55,18 @@ def alignment_flow(feature_net, alignment_cache_path=None):
     alignment = net.add_node(
         "speech-alignment",
         "alignment",
-        {"id": "$(id)", "orthography": "$(orthography)"},
+        {
+            "id": "$(id)",
+            "orthography": "$(orthography)",
+        },
     )
+    if add_left_right_context_orth:
+        net.nodes["alignment"].update(
```

Comment:
```suggestion
        net.nodes[alignment].update(
```
`add_node` might rename your node if a node with the same name already exists. Thus you should always use the name returned by `add_node` when modifying the node afterwards.